### PR TITLE
added walk-outside story

### PR DESCRIPTION
**Pull request by rypersaud**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/1701)

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: english/marshmallow.md [hunk 1]
```diff
@@ -306,3 +306,5 @@ Do you:
 [Watch Up](watch-movie/Up.md)
 
 [Give up and search for oil fields](oil-field/oil-field.md)
+
+[Walk outside](walk-outside/walkoutside.md)
```
Comment (casey-collab) [inline english/marshmallow.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/walk-outside.md [hunk 2]
```diff
@@ -0,0 +1 @@
+You walk outside and it is a beautiful day
```
Comment (casey-collab) [inline english/walk-outside.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

Comment (casey-collab) [inline english/walk-outside.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**